cmd/milmove/gen_duty_stations_migration.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+
+	"github.com/transcom/mymove/pkg/cli"
+	"github.com/transcom/mymove/pkg/logging"
+	dutyStations "github.com/transcom/mymove/pkg/services/duty_stations"
+)
+
+const (
+	// filename containing the details for new office users
+	DutyStationsFilenameFlag string = "office-users-filename"
+)
+
+type MigrationInfo struct {
+	Filename string
+}
+
+const (
+	DutyStationMigration string = `
+-- Migration generated using: cmd/milmove/gen_duty_stations_migration.go
+-- Duty stations file: {{.Filename}}`
+)
+
+// DutyStationsFilenameFlag initializes add_office_users command line flags
+func InitAddDutyStationsFlags(flag *pflag.FlagSet) {
+	flag.StringP(DutyStationsFilenameFlag, "f", "", "File name of csv file containing the new office users")
+}
+
+// CheckAddDutyStations validates add_office_users command line flags
+func CheckAddDutyStations(v *viper.Viper) error {
```

Comment:
You need this code from:

- https://github.com/transcom/mymove/blob/master/cmd/milmove/migrate.go#L52-L54
- https://github.com/transcom/mymove/blob/master/cmd/milmove/gen_office_user_migration.go#L59-L65

```suggestion
func CheckAddDutyStations(v *viper.Viper) error {

	if err := cli.CheckDatabase(v, logger); err != nil {
		return err
	}
	
	if err := cli.CheckMigration(v); err != nil {
		return err
	}

	if err := cli.CheckMigrationFile(v); err != nil {
		return err
	}

```